gr-filter/python/filter/design/fir_design.py

```diff
@@ -27,10 +27,14 @@ def design_win_lpf(fs, gain, wintype, mainwin):
         try:
             taps = filter.firdes.low_pass_2(gain, fs, pb, tb,
                                             atten, wintype)
-        except RuntimeError as e:
+        except RuntimeError  as e:
             reply = QtGui.QMessageBox.information(mainwin, "Runtime Error",
                                                   e.args[0], QtGui.QMessageBox.Ok)
             return ([], [], ret)
+        except IndexError  as e:
+            reply = QtGui.QMessageBox.information(mainwin, "Runtime Error",
+                                                  e.args[0], QtGui.QMessageBox.Ok)
+            return ([], [], ret)            
```

Comment:
```suggestion
        except (RuntimeError, IndexError) as e:
            reply = QtGui.QMessageBox.information(mainwin, "Runtime Error",
                                                  e.args[0], QtGui.QMessageBox.Ok)
            return ([], [], ret)
```
Alternatively, have different error messages in the message box.